pysite/views/main/index.py

```diff
@@ -1,10 +1,12 @@
 # coding=utf-8
 from pysite.base_route import RouteView
+from pysite.constants import DISCORD_OAUTH_REDIRECT
+from pysite.mixins import OauthMixin
 
 
-class IndexView(RouteView):
+class IndexView(OauthMixin, RouteView):
     path = "/"
     name = "index"
 
     def get(self):
-        return self.render("main/index.html")
+        return self.render("main/index.html", logged_in=self.logged_in, login_url=DISCORD_OAUTH_REDIRECT)
```

Comment:
`logged_in` is required on all pages. This parameter should be added in the `render()` method of the base class instead. I've already got one somewhere.

Reply:
Also, we should have a login url ourselves that redirects to the Discord OAuth page.